include/sys/dsl_scan.h

```diff
@@ -55,6 +56,7 @@ typedef struct dsl_scan_phys {
 	uint64_t scn_cur_max_txg;
 	uint64_t scn_start_time;
 	uint64_t scn_end_time;
+	uint64_t scn_scrub_pause_time; /* when scrub was paused */
```

Comment:
yea I realized that after @loli10K's comment about errata. What I'm going to have to do avoid changing on disk format is have the pause time reset every time the pool is imported in a paused state which I think is ok.